packages/controller/io-package.json

```diff
@@ -932,6 +932,36 @@
           },
           "regex": [],
           "limit": 1
+        },
+        {
+          "category": "blocked",
+          "name": {
+            "en": "Instance was deactivated",
+            "de": "Instanz wurde deaktiviert",
+            "ru": "Экземпляр был деактивирован",
+            "pt": "A instância foi desativada",
+            "nl": "Instruatie",
+            "fr": "Instance was deactivd",
+            "it": "Il ricorso è stato disattivato",
+            "es": "Se desactiva la instalación",
+            "pl": "Instytucja została deaktywowana",
+            "zh-cn": "案件"
+          },
+          "severity": "alert",
+          "description": {
+            "en": "Installed version was blocked by ioBroker team.\nPlease update adapter to newer or available version before enabling it.\nInstance must be enabled manually.",
```

Comment:
`to newer or available`? Do you mean to latest or stable?

Reply:
If the actual version is blocked and it is blacklisted, so no newer version exists. But yes we can write stable or latest version.
May be `to official version from repository` ?

Reply:
Can we guide the user a bit better here?
* If a newer, non-blacklisted version exist, propose that:
  `...was blacklisted by the ioBroker team. Please upgrade to version x.y.z and ...`
* Else find the last non-blacklisted version and propose that:
  `...was blacklisted by the ioBroker team. Please downgrade to version x.y.z and ...`
* If the blocked version is the only one:
  `...was blacklisted by the ioBroker team, but no other version exists. You need to wait until a fixed version is released.`

Reply:
x.y.z is not that easy ... unless we state "at least x.y.z"

Reply:
> Can we guide the user a bit better here?
> 
> * If a newer, non-blacklisted version exist, propose that:
>   `...was blacklisted by the ioBroker team. Please upgrade to version x.y.z and ...`
> * Else find the last non-blacklisted version and propose that:
>   `...was blacklisted by the ioBroker team. Please downgrade to version x.y.z and ...`
> * If the blocked version is the only one:
>   `...was blacklisted by the ioBroker team, but no other version exists. You need to wait until a fixed version is released.`

How should we add it all to description?

Reply:
Current implementation: https://github.com/ioBroker/ioBroker.admin/blob/master/main.js#L1062
Here it is possible, but later it will be moved into js-controller